airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -2,21 +2,24 @@
 
 from contextlib import contextmanager
 
+from airflow.configuration import conf
 from airflow.models import DagModel, DagRun, TaskInstance, TaskFail, XCom
 from airflow.plugins_manager import AirflowPlugin
 from airflow.settings import Session
 from airflow.utils.state import State
+from airflow.utils.log.logging_mixin import LoggingMixin
+from airflow_prometheus_exporter.xcom_config import xcom_config
 from flask import Response
 from flask_admin import BaseView, expose
+import json
+import pickle
```

Comment:
We try to follow pep8 import order (https://pep8.org/#imports) so we can add these two imports to the job